File: Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/repositories/models/response/AnswersResponse.java
package com.dreams.interviewprepapp.repositories.models.response;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Tc2r on 5/19/2017.
 * <p>
 * Description:
 */

public class AnswersResponse {
    @SerializedName("answers")
    private ArrayList<Answer> answersList;

    public ArrayList<Answer> getAnswersList() {
        return answersList;
    }

}

File: Interview_Prep_App/native_kotlin/view_binding/src/main/java/com/interviewprep/kotlinviewbinding/repository/models/response/AnswersResponse.kt
package com.interviewprep.kotlinviewbinding.repository.models.response

import com.google.gson.annotations.SerializedName
import com.interviewprep.kotlinviewbinding.repository.models.Answer

data class AnswersResponse(
    @SerializedName("answers")
    var answers: ArrayList<Answer>
)
